examples/bri-3/src/bri/workgroup/workflows/api/workflows.controller.spec.ts

```diff
@@ -52,9 +58,15 @@ describe('WorkflowsController', () => {
   };
 
   beforeEach(async () => {
-    mockWorkstepStorageAgent = new MockWorkstepStorageAgent();
+    mockWorkstepStorageAgent = new MockWorkstepStorageAgent(mapper);
```

Comment:
either pull MockWorkflowStorageAgent initialization here or move this one bellow to useValue